src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileStatusService.scala

```diff
@@ -25,6 +26,19 @@ class FileStatusService(fileStatusRepository: FileStatusRepository, uuidSource:
     fileStatusRepository.addFileStatuses(rows).map(_.map(row => FileStatus(row.fileid, row.statustype, row.value)).toList)
   }
 
+  def getConsignmentFileProgress(consignmentId: UUID): Future[FileChecks] = {
```

Comment:
Should add some unit tests for this new function

Reply:
I've added some in, thanks.

Reply:
I've added some in, thanks.